siac/walletcmd.go

```diff
@@ -373,6 +381,61 @@ Siafund Claims:      %v SC
 `, encStatus, lockStatus, sc, usc, status.ConfirmedSiacoinBalance, status.SiafundBalance, status.SiacoinClaimBalance)
 }
 
+// wallettransactionscmd lists all of the transactions related to the wallet,
+// providing a net flow of siacoins and siafunds for each.
+func wallettransactionscmd() {
+	wtg := new(api.WalletTransactionsGET)
+	err := getAPI("/wallet/transactions?startheight=0&endheight=10000000", wtg)
+	if err != nil {
+		fmt.Println("Could not fetch transaction history:", err)
+		return
+	}
+
+	fmt.Println("[height]\t[transaction id]\t\t\t\t\t\t\t[net siacoin transfer]\t[net siafund transfer]")
```

Comment:
maybe I should just not worry about formatting the first line until we can get a more consistent unit